repo2module/tests/test_repo2module.py

```diff
@@ -1,9 +1,6 @@
 import unittest
 import os.path
-
 import createrepo_c
-from click.testing import CliRunner
```

Comment:
Nitpick: There is a (skipped) test that uses this class.

Reply:
True, thank you for pointing that out. I wasn't sure what to do with that call. I'll probably comment-out the code to have some example for when somebody decides to un-skip that test.